Boundary Docs (#2907)
hashicorp/web-unified-docs: content/boundary/v1.0.x/content/docs/concepts/workers.mdx

---
layout: docs
page_title: Boundary workers
description: >-

>   Workers let you proxy traffic to private targets. Learn about worker capabilities, multi-hop sessions, tags for filtering, and how to monitor worker status and health.

Reviewer: ```suggestion
  Workers let you proxy traffic to private targets. Learn about worker capabilities, multi-hop sessions, tags for filtering, and monitoring status and health.
```

# START AUTO GENERATED METADATA, DO NOT EDIT
created_at: 2025-11-18T17:04:41-05:00
last_modified: 2026-01-26T14:35:46-05:00
# END AUTO GENERATED METADATA
---

> # Boundary workers

Reviewer: ```suggestion
# Workers
```

inbound traffic to route through multiple network enclaves in order to reach the target system.
Multi-hop sessions allow you to chain together two or more workers across multiple networks to form reverse proxy connections between the user and the target, even in complex networks with strict outbound-only policies.

> ## Filter workers with tags

Reviewer: ```suggestion
## Tags
```

Reviewer: I changed this one because when the heading starts with a verb, I think the user expectation is that it will be a how-to/procedural doc. But this section is just conceptual.
